calamari-core/src/main/java/org/starchartlabs/chronicler/calamari/core/MediaTypes.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) Oct 1, 2018 StarChart Labs Authors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    romeara - initial API and implementation and/or initial documentation
+ */
+package org.starchartlabs.chronicler.calamari.core;
+
+//TODO romeara
```

Comment:
Shall doc - just bear in mind this is just a step of the WIP conversion, wanted to get it in now that it works end-to-end (more than the old thing did) and start working on specific, smaller tasks instead of a giant feature branch

Reply:
Documentation added